packages/foreman/foreman/foreman.spec

```diff
@@ -34,6 +34,8 @@ BuildArch:  noarch
 Conflicts: foreman-tasks < 0.11.0-2
 Conflicts: foreman-release-scl < 7-1
 
+Obsoletes: foreman-compute
```

Comment:
Should this be
```suggestion
Obsoletes: foreman-compute < 1.22.0-0.9.develop
```

Reply:
or `< %{version}`

Reply:
```
10:31 < ewoud> Zhenech: I think your suggestion is correct
10:33 < ewoud> either can work, but I think the latter one is a bit more robust
10:34 < ewoud> the explicit version number does show that at some point you could drop it more easily, but we don't really do that anyway
```

Reply:
amended. in the future feel free to push to my branch, especially when i'm afk :)